plugins/base/src/main/kotlin/renderers/html/HtmlRenderer.kt

```diff
@@ -719,8 +719,9 @@ open class HtmlRenderer(
 
     private fun resolveLink(link: String, page: PageNode): String = if (URI(link).isAbsolute) link else page.root(link)
 
-    open fun buildHtml(page: PageNode, resources: List<String>, content: FlowContent.() -> Unit) =
-        createHTML().prepareForTemplates().html {
+    open fun buildHtml(page: PageNode, resources: List<String>, content: FlowContent.() -> Unit): String {
+        val pathToRoot = locationProvider.pathToRoot(page)
```

Comment:
Please inline or use it consistently for all usages (there is a second `pathToRoot` call in the scope)